mobile/lib/main.dart

```diff
@@ -83,92 +85,116 @@ class TenTenOneApp extends StatefulWidget {
 }
 
 class _TenTenOneAppState extends State<TenTenOneApp> {
+  bool showOnboarding = false;
+
   final GoRouter _router = GoRouter(
-    navigatorKey: _rootNavigatorKey,
-    initialLocation: WalletScreen.route,
-    routes: <RouteBase>[
-      ShellRoute(
-        navigatorKey: _shellNavigatorKey,
-        builder: (BuildContext context, GoRouterState state, Widget child) {
-          return ScaffoldWithNavBar(child: child);
-        },
-        routes: <RouteBase>[
-          GoRoute(
-            path: WalletScreen.route,
-            builder: (BuildContext context, GoRouterState state) {
-              return const WalletScreen();
-            },
-            routes: <RouteBase>[
-              GoRoute(
-                path: SendScreen.subRouteName,
-                // Use root navigator so the screen overlays the application shell
-                parentNavigatorKey: _rootNavigatorKey,
-                builder: (BuildContext context, GoRouterState state) {
-                  return const SendScreen();
-                },
-              ),
-              GoRoute(
-                path: SeedScreen.subRouteName,
-                // Use root navigator so the screen overlays the application shell
-                parentNavigatorKey: _rootNavigatorKey,
+      navigatorKey: _rootNavigatorKey,
+      initialLocation: WelcomeScreen.route,
+      routes: <RouteBase>[
```

Comment:
I have no idea why suddenly it formatted with 4 spaces instead of 2.... @bonomat is this the problem you had too recently?

Reply:
No, I haven't had this problem yet 😬